libs/sdk-backend-bear/src/backend/workspace/accessControl/index.ts

```diff
@@ -32,4 +55,44 @@ export class BearWorkspaceAccessControlService implements IWorkspaceAccessContro
         );
         return this.authCall((sdk) => sdk.project.removeGrantees(objectUri, granteeUris));
     }
+
+    public async changeAccess(
+        sharedObject: ObjRef,
+        grantees: GranteeWithGranularPermissions[],
+    ): Promise<void> {
+        // change access for grantees with some permissions provided
```

Comment:
What about not implementing this function on Bear? Do we need it to be implemented?

Reply:
We could skip it, but then we would have to introduce additional logic to the UI component. It is more convenient and straightforward to implement the same SPI method and the UI component just calls it as backend-agnostic.

Reply:
OK, then let's keep it. What about little bit explaining that on Bear "some permissions -> grant access", "no permissions -> revoke access" the TSDoc way. and BTW should VIEW permission result in granting access?

Reply:
Will add some doc. 

Yes, any permission means granting access as there is no granularity on Bear. You either have permission or you don't AFAIK.